content/en/armory-enterprise/quickspin/quickstart-quick-spin.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Quick Spin Quick Start 
+linkTitle: "Start using Quick Spin"
+no_list: true
+weight: 5
+description: >
+  Use this tutorial to lean how to create and manage a simple Spinnaker workflow.
+exclude_search: true
+---
+## Abstract
+This tutorial demonstrates how to:
+- Create a Spinnaker application
+- Create a Jenkins instance and add it to your application
+- Deploy and manage a Spinnaker pipeline for your application
+- Deploy updates to the application
+- Monitor the health of your application
+
+## Prerequisites
+Before you begin make sure you have installed Quick Spin and have an active instance running on `locahost:9000` as described in: TBD.
+
+- You need an active github account
+- Your favorite code editor with some JavaScript support enabled
+- 
+
+## Run the Spinnaker prep pipeline
+
+## Create kub cluster
+
+## Get the tutorial repository:
+1. From your browser navigate to:
+   ```
+   https://github.com/arnab-datta/counter-app
```

Comment:
Great that you found this sample app. If you want to use Armory CD to deploy this sample app, you should fork it into github.com/armory and then reference that repo (so we own the repo).  
Are you going to "dockerize" this app and create K8s manifest files?

Reply:
Will get back to this one tomorrow